docs/content/dagster-cloud/deployment/agents/customizing-configuration.mdx

```diff
@@ -94,3 +94,47 @@ compute_logs:
   module: dagster.core.storage.noop_compute_log_manager
   class: NoOpComputeLogManager
 ```
+
+---
+
+## Writing compute logs to AWS S3
+
+You can write compute logs to an AWS S3 bucket without requiring your S3 credentials by configuring the [dagster_aws.s3.compute_log_manager](https://docs.dagster.io/\_apidocs/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager) module.
```

Comment:
"without requiring your s3 credentials" isn't quite right - you still need to provide s3 credentials in the dagster.yaml file, you just don't need to provide them to Dagster Labs (and we have no capability for you to do so, even if you wanted to)

Reply:
Gotcha, how about this verbiage in the form of a note?

You can write compute logs to an AWS S3 bucket by configuring the dagster_aws.s3.compute_log_manager module.

You are also able to stream partial compute log files by configuring the log upload interval (in seconds) using the `upload_interval` parameter.

Note: Dagster Labs will neither have nor use your AWS credentials. The Dagster Cloud UI will be able to show the URLs linking to the compute log files in your S3 bucket when you set the `show_url_only` parameter to `true`.